spec/api-web-contents-spec.js

```diff
@@ -1273,4 +1273,43 @@ describe('webContents module', () => {
       expect(data).to.be.an.instanceof(Buffer).that.is.not.empty()
     })
   })
+
+  // FIXME: disable during chromium update due to crash in content::WorkerScriptFetchInitation::CreateScriptLoaderOnIO
+  xdescribe('Shared Workers', () => {
+    let worker1
+    let worker2
+    let vectorOfWorkers
+
+    before(() => {
+      worker1 = new SharedWorker('../fixtures/api/shared-worker/shared-worker1.js')
+      worker2 = new SharedWorker('../fixtures/api/shared-worker/shared-worker2.js')
+      worker1.port.start()
+      worker2.port.start()
+    })
+
+    after(() => {
+      worker1.port.close()
+      worker2.port.close()
+    })
+
+    it('can get multiple shared workers', () => {
+      const contents = webContents.getAllWebContents()
```

Comment:
Please use `remote.getCurrentWebContents()` to get the current `WebContents` instead of `contents[0]`, which can be any other `WebContents` instance.